scripts/application.d/solvcon

```diff
@@ -14,8 +14,8 @@ mkdir -p ${DEVENVAPP}
 
 ln -s ${application_src} ${DEVENVAPP}/${application_name}
 
-
-echo "launching solvcon..."
-${DEVENVROOT}/bin/build-application-solvcon-devenv.sh
+# devenv scripts to build SOLVCON application
+DEVENVAPPBUILDSRC=${DEVENVROOT}/applications/solvcon
+source ${DEVENVAPPBUILDSRC}/entry.sh
```

Comment:
Using a naming convention `DEVENVAPPBUILDSRC` and `entry.sh`. The style of this naming convention will be more generic and followed by the other app build scripts in the future.